docs/2.7.0/docs/daml-finance/tutorials/settlement/transfer.rst

```diff
@@ -0,0 +1,152 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Transfer
+########
+
+In this tutorial, we delve deeper into the concepts that were introduced in our getting-started
+tutorials. In particular, we will extend on the :ref:`transfer <../getting-started/transfer>`
+tutorial.
+
+We begin by understanding the simple form of settlement. It transpires when a customer's funds are
+transferred to another account within the same bank. Consequently, the sender's account is debited
+(balance decreases), and the recipient's account is credited (balance increases). This process is
+internally managed within the bank's systems and usually occurs instantly, as it doesn't require
+interaction with external systems or institutions.
+
+In Daml Finance, such fund transfers are not necessarily represented by a settlement workflow that
+involves allocating and approving instructions. Instead, a "direct" transfer of funds can occur
+between two parties, such as Alice and Bob. This transfer debits the sending account and atomically
+credits the receiving account.
+
+Next, we'll explore how to configure controllers responsible for authorizing incoming and outgoing
```

Comment:
Given that you already introduce the notion of `Debit` and `Credit`, would it be useful to rephrase as ... the controllers responsible for authorizing debits and credits of holdings in an account. Otherwise it's not necessarily clear that an outgoing transfer is a debit and an incoming transfer is a credit

Reply:
agree, will adjust